src/storage-types/src/parameters.proto

```diff
@@ -47,9 +48,16 @@ message ProtoPgSourceTcpTimeouts {
     optional mz_proto.ProtoDuration tcp_user_timeout = 5;
 }
 
+message ProtoKafkaSourceTcpTimeouts {
+    bool keepalive = 1;
+    mz_proto.ProtoDuration socket_timeout = 2;
+    mz_proto.ProtoDuration transaction_timeout = 5;
+    mz_proto.ProtoDuration socket_connection_setup_timeout = 3;
```

Comment:
In the pg and ssh settings we call this `connect_timeout`, which is I think the standard name for this setting. Can we use the same name for both? Or is it actually different?

Reply:
these names are based directly on the kafka setting names